cmd/agent/container/setup.go

```diff
@@ -200,6 +206,74 @@ func (cmd *SetupContainerCmd) Run(ctx context.Context) error {
 	return nil
 }
 
+func cloneRepository(ctx context.Context, workspaceInfo *provider2.ContainerWorkspaceInfo, helper string, log log.Logger) error {
```

Comment:
This is shared with cmd/agent/workspace/up, right? Could we consolidate it or doesn't it make sense right now?